Update text_form_field.dart

guihcs opened the pull request:
updated maxLength assertion to coincide with TextField maxLength asserion.

Changed maxLength assertion to include TextField.noMaxLength in TextFormField

* TextFormField maxLength assertion is evaluating to false when TextField.noMaxLenght value is used.

guihcs committed:
Update text_form_field.dart

updated maxLength assertion to coincide with TextField maxLength asserion.

guihcs commented:
@googlebot I signed it!

LongCatIsLooong commented:
Thanks for the PR! The change makes sense to me. Could you add a test to "flutter/packages/flutter/test/material/text_form_field_test.dart" to make sure this does not regress? Something like this but simpler:

https://github.com/flutter/flutter/blob/3cbfe82d9d35ecd0b27bfd0b7ffb887787715f60/packages/flutter/test/widgets/editable_text_test.dart#L6563-L6589

guihcs committed:
Update text_form_field_test.dart

Added test to verify that TextFormField accepts TextField.noMaxLength as value to maxLength parameter.

guihcs committed:
Update text_form_field_test.dart

LongCatIsLooong commented on packages/flutter/test/material/text_form_field_test.dart:
```diff
@@ -113,7 +113,20 @@ void main() {
     expect(find.byType(CupertinoButton), findsNothing);
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.macOS, TargetPlatform.windows, TargetPlatform.linux }), skip: kIsWeb);
-
+  
```
There are empty spaces on line 116 and 129. Could you remove them?

guihcs committed:
Update text_form_field_test.dart
